.circleci/workflows.yml

```diff
@@ -331,16 +382,37 @@ jobs:
           command: bash docker/scripts/test-container-image.sh ${IMAGE_NAME}:sha-$(git rev-parse HEAD)
       - when:
           condition:
-             equal: [ main, <<pipeline.git.branch>> ]
+            not: 
+              equal: [ main, <<pipeline.git.branch>> ]
```

Comment:
Why this change?

Reply:
Somehow fucked it up without noticing.

Reply:
Ah, wait. I thought it was somewhere. This must be correct. This is the dev release. So it only happens on branches which are not main, and only if anything in the artifacts has changed.